console/console-init/ui/src/modules/iot-device/components/DeviceListToolbar/DeviceListToolbar.tsx

```diff
@@ -31,51 +37,59 @@ export const DeviceListToolbar: React.FunctionComponent<IDeviceListToolbarProps
   handleJSONUpload,
   onSelectAllDevices,
   isChecked,
-  kebabItems
+  kebabItems,
+  handleToggleModal
 }) => {
   return (
-    <Toolbar id="device-data-toolbar" data-codemods="true">
-      <ToolbarContent id="device-data-toolbar-content">
-        <ToolbarItem
-          variant="bulk-select"
-          id="device-list-toolbar-item-1"
-          key="bulk-select"
-          aria-label="Select multiple devices"
-          data-codemods="true"
-        >
-          <DropdownWithBulkSelect
-            dropdownId="device-bulk-select"
-            dropdownToggleId="device-bulk-select-toggle"
-            checkBoxId="device-bulk-select-checkbox"
-            ariaLabel="Bulk select dropdown for device list"
-            isChecked={isChecked}
-            onChange={onSelectAllDevices}
-          />
-        </ToolbarItem>
-        <ToolbarItem
-          id="device-list-toolbar-item-2"
-          key="create-device"
-          aria-label="Create device button"
-          data-codemods="true"
-        >
-          <CreateDeviceButton
-            handleInputDeviceInfo={handleInputDeviceInfo}
-            handleJSONUpload={handleJSONUpload}
-          />
-        </ToolbarItem>
-        <ToolbarItem
-          id="device-list-toolbar-item-3"
-          key="kebab-dropdown"
-          aria-label="Device list kebab dropdown"
-          data-codemods="true"
-        >
-          <DropdownWithKebabToggle
-            isPlain={true}
-            dropdownItems={kebabItems}
-            id="device-list-toolbar-kebab"
-          />
-        </ToolbarItem>
-      </ToolbarContent>
-    </Toolbar>
+    <>
+      <Toolbar id="device-data-toolbar" data-codemods="true">
+        <ToolbarContent id="device-data-toolbar-content">
+          <ToolbarItem
+            variant="bulk-select"
+            id="device-list-toolbar-item-1"
```

Comment:
This is not really nice ID. I would rather don't use number in ID's because that can way to some stupid naming like button-1, button-2 etc... Add some intelligent ID here :)